crates/iota/src/name_commands.rs

```diff
@@ -65,7 +65,7 @@ pub enum NameCommand {
     #[command(subcommand)]
     Auction(AuctionCommand),
     /// Check the availability of a name and return its price if available.
-    /// Subnames are always free to register by the parent name owner.
+    /// Subnames are always free to register by the parent name owner
```

Comment:
Why are you removing this dot but not the previous one?

Reply:
Is it what we are actually doing?

Reply:
https://github.com/iotaledger/iota/pull/7845/commits/85010fd7449c40ff2e0e3128b643023325a2e038